src/promptflow/tests/executor/e2etests/test_executor_validation.py

```diff
@@ -289,11 +314,12 @@ def test_single_node_input_type_invalid(
         # Single Node run - the inputs are from flow_inputs + dependency_nodes_outputs
         with pytest.raises(error_class) as exe_info:
             FlowExecutor.load_and_exec_node(
-                flow_file=get_yaml_file(flow_folder, path_root),
+                flow_file="flow.dag.yaml",
```

Comment:
there should be a common constant iirc, try use that

Reply:
good point